tutorials/i18n/internationalizing_games.rst

```diff
@@ -58,6 +58,41 @@ Select the resource to be remapped then add some alternatives for each locale.
     the current language, making it ideal for things like multiplayer chat where
     the text language may not match the client's language.
 
+Automatically setting a language
+--------------------------------
+It is recommended to default to the user's preferred language which can be obtained via :ref:`OS.get_locale_language() <class_OS_public_method_get_locale_language>`. If your game is not available in that language, it will fallback to the one in **Project settings > General > Internationalization > Locale (Advanced settings)** (if empty `en` for English will be used which is recommended).
```

Comment:
```suggestion
It is recommended to default to the user's preferred language which can be obtained via :ref:`OS.get_locale_language() <class_OS_public_method_get_locale_language>`. If your game is not available in that language, it will fall back to the one in **Project settings > General > Internationalization > Locale (Advanced settings)** (if empty `en` for English will be used which is recommended).
```